cfgov/v1/models/snippets.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+from wagtail.wagtailsnippets.models import register_snippet
+from wagtail.wagtailcore.fields import RichTextField, StreamField
+from wagtail.wagtailadmin.edit_handlers import FieldPanel, StreamFieldPanel
+
+from . import molecules
+
+
+@register_snippet
+class Contact(models.Model):
+    heading = models.CharField(max_length=22, blank=False)
```

Comment:
So, my thinking was that the you use to define `convert` would also define a `get_existing_snippet` because of this exact problem. There wasn't a clear way of distinguishing one snippet from another, so you had to tell it how to find them uniquely.

At this point, I'm not in favor of making a base class just yet.